website/src/main/java/com/google/step/data/ClassHandler.java

```diff
@@ -40,4 +40,10 @@ default ClassifyTextRequest convertDocToRequest(Document doc) {
   default Document convertStringToDoc(String text) {
     return Document.newBuilder().setContent(text).setType(Type.PLAIN_TEXT).build();
   }
+
+  public static ClassifyTextResponse getTestCategory() {
+    ClassificationCategory c = ClassificationCategory.newBuilder().build()
+        .toBuilder().setName("/Fake/Fake1/Fake2/Fake3").build();
```

Comment:
Please remove test categories for final product, or separate test/prod environments

Reply:
A separate environment for production is coming soon